web-src/src/locales/en.json

```diff
@@ -471,9 +471,8 @@
         },
         "spotify": {
           "no-support": "OwnTone was either built without support for Spotify or libspotify is not installed.",
-          "help": "If you normally log into Spotify with your Facebook account you must first go to Spotify's web site where you can get the Spotify username and password that matches your account.",
           "logged-as": "Logged in as ",
-          "requirements": "You must have a Spotify premium account.",
```

Comment:
Please remove the additional text for the "requirements". 
The help text for Spotify with Facebook, etc. login is not relevant any more (since moving away from libspotify).

Reply:
Excellent. It has been modified accordingly.